schema.graphql

```diff
@@ -52,6 +53,7 @@ type Unipool @entity {
 
 type TokenDistroBalance @entity {
   id: ID!
+  user: User!
```

Comment:
Here token distro address should be saved too, like tokenBalance and unipoolBalance.